### PR TITLE
Implemented fix from @ifyouseewendy

### PR DESCRIPTION
**Pull request by aspidites**
# Description
This basically implements the fix from #1424. The only deviation from the proposed fix is to imply the '--write-mode inplace' option as I cannot imagine a case where you don't want that, and I don't think it's idiomatic for a user to have to specify that in there config before the fixer even works. 

I have to admit that I've never written VimL, so don't know how to use vader. Looking in `test/fixers`, though, it appears that we are only testing that the command is called correctly, not that a particular fixer executes as expected, so I think I'm OK? I've also never used vint, but after installing `vint -s brittany.vim` doesn't produce output, so I assume I'm OK here as well. 

Happy to take feedback in either case, so long as you realize that responses will be a bit slow, as I only really have weekends to play around with this.

### CODE REVIEW

File: autoload/ale/fixers/brittany.vim [hunk 1]
```diff
@@ -1,13 +1,17 @@
-" Author: eborden <evan@evan-borden.com>
+" Author: eborden <evan@evan-borden.com>, ifyouseewendy <ifyouseewendy@gmail.com>, aspidiets <emarshall85@gmail.com>
 " Description: Integration of brittany with ALE.
 
 call ale#Set('haskell_brittany_executable', 'brittany')
+call ale#Set('haskell_brittany_options', '--write-mode inplace')
```
Comment (w0rp) [inline autoload/ale/fixers/brittany.vim]: Does brittany need this option to be set to work? If so, take this out of the options and write it into the command string below. Was this always needed, and did the brittany fixer never work, or did the behavior change in a recent version? If the behavior did change, then we probably detect which version of brittany is being run, and only add this option if needed.

**Comment by w0rp** [comment]
The tests need to be updated here to reflect your changes. I'll sort that out.

**Comment by w0rp** [comment]
Also, there was a space missing before the argument.

**Comment by w0rp** [comment]
I have included this and fixed it now.

**Comment by aspidites** [comment]
@w0rp , I learned that the hard way. Didn't realize '.' was concatenation until 2 minutes ago. Went ahead and squashed changes as well to prevent things from looking absurd in the commit history. I really ought to learn how to test things locally.

**Comment by aspidites** [comment]
@w0rp, you're way too fast :laughing:. I was hoping that you wouldn't notice my fumbling about vimscript.

**Pull request closed without merging**